src/clj/muuntaja/json.clj

```diff
@@ -116,13 +126,34 @@
      (.readValue mapper ^String data ^Class Object)
      (.readValue mapper ^InputStream data ^Class Object))))
 
+(defn from-json-with-opts
```

Comment:
Could the `from-json` & `to-json` just check the argument type to see if it was given opts or mapper?